index.html

```diff
@@ -5,16 +5,83 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/grid.css"> 
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
-
 <body>
+<div class="container">
+    <div class="row"> <!--header-->
+        <header>
+            <div class="col-6">
+            <h1 class="logo-title">ALLAN JOHNSON</h1>
+            <h3 class="logo-sub">Personal Life Coach</h3>
+        </div>
+            <nav>
+                <div class="col-6">
+                <ul>
+                    <li>About</li>
+                    <li>Services</li>
+                    <li>Motivational Reading</li>
+                    <li>Blog</li>
+                    <li>Contact</li>
+                    <li>Log In</li>
+               </ul>
+        </nav>
+        </header>
+    </div>
 
-    <!-- Add your markup here -->
+    <div class="row"> <!--hero-->
+        <div class="col-12 hero">
+            <p class="ambition">Ambition is the first step towards</p>
+            <p class="success">SUCCESS</p>
+            <button class="book-now" type="button">Book Now</button>
+        </div>
+    </div>
+    <div class="row"> <!--About Me-->
+       <div class="row3">
+        <div class="col-6 left">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur, accusamus ut veniam ratione amet tempore minus expedita eum corporis nihil ducimus laudantium optio. Voluptatum unde error iusto minus repellendus animi.</p>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores facere enim quod porro minus. Deleniti, quasi. Sint repellendus ipsam reiciendis. Facere, tempora alias sint eos ex atque voluptate aspernatur delectus.</p>
+            <button class="read-more" type="button">Read More</button>
+        </div> 
+       <div class="col-6 right"></div>
+        </div>
+    </div>
+    <div class="row"> <!--How I Can Help-->
+        <div class="col-12 help">
+            <h1>How I Can <br>Help You</h1>
+        <div class="coaching">
+            <h1 class="box">ONE ON ONE COACHING</h1>
+            <p class="coaching-info">1 hr<br>£40</p>
+            <button class="book-it" type="button">Book It</button>
+        </div>
+    </div>
+    </div>
```

Comment:
Watch your indentation, it makes it hard to follow.

Reply:
This 'How can I help' section is also a good candidate for what I mentioned above: once you've removed the overall `container`, you can add a `container` inside this section, after the background images has been set (which should be full-width), and before the `row` and the `col-12`.
(see code snippet shared above)